libdevcore/CommonIO.cpp

```diff
@@ -126,6 +129,29 @@ std::string dev::getPassword(std::string const& _prompt)
 	std::getline(cin, ret);
 	return ret;
 #else
-	return getpass(_prompt.c_str());
+	struct termios oflags;
+	struct termios nflags;
+	char password[128];
```

Comment:
Yeah ofc. The reason for choosing 128 is that this is the internal buffer limit that `getpass()` actually used if you check out the documentation I linked above.